.github/workflows/codeql-analysis.yml

```diff
@@ -9,7 +9,7 @@ on:
   push:
   pull_request:
   schedule:
-    - cron: '0 13 * * 6'
+    - cron: '0 9 * * 1'
```

Comment:
Is this in UTC? 😛

Reply:
Timezones 🤦  yes it is 
https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows